index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Glory Mugizi Portfolio</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul class="tool-bar">
+          <li class="my-logo">MY LOGO</li>
+          <li class="toggle-menu">
+            <img src="images/humbger.png" alt="" srcset="" />
+          </li>
+        </ul>
+      </nav>
+      <section class="headline">
+        <div class="container heading">
+          <h1>I’m Glory Mugizi Glad to see you!</h1>
+        </div>
+        <div class="container paragraph-one">
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <a href="" class="lets-connect">LET’S CONNECT</a>
```

Comment:
- [ ] This isn't a link so I suggest you use a text tag like `p` instead of `a` tag 👍

Reply:
Hi @ha-manel,
a tag was replaced with p tag as requested.
Thank you.